external-api/src/v1/controllers/party-db.controller.ts

```diff
@@ -25,7 +25,57 @@ export const lookup = async (req: Request, res: Response) => {
     headers,
   }).catch((error: AxiosError) => {
     console.error('Error calling Party DB API %o', error);
-    return { data: 'Failed to call Party DB API', status: error?.response?.status || HttpStatusCode.InternalServerError };
+    return { data: 'Error calling Party DB API', status: error?.response?.status || HttpStatusCode.InternalServerError };
+  });
+
+  const { status, data } = response;
```

Comment:
I would suggest wrapping the `lookup` function to `try-catch` and remove axios catch please. This will negate the need of validating `response object at LOC:31.

Reply:
Thank prefer the `try-catch` thank you for adding.